activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Set polymorphic `*_type` column to NULL on `dependent: :nullify` strategy. 
+    
+    On polymorphic associations both the `foreign key` and the `*_type `column will be set to NULL.
+    
```

Comment:
Sign your name at the end of the changelog message like this:

```
*   This is a changelog entry.

    *FirstName LastName*
```
(see previous entries for examples, you may also use your github username instead of your real name)